server/services/utils/permissions/getAdjudicationsPermissions.test.ts

```diff
@@ -0,0 +1,24 @@
+import { prisonerMDI, userLEI, userMDI } from '../../../../tests/mocks/userMocks'
+import { Role } from '../../../data/enums/role'
+import getAdjudicationsPermissions from './getAdjudicationsPermissions'
+
+const caseLoadListWithMdi = [
+  { caseLoadId: 'MDI', currentlyActive: false },
+  { caseLoadId: 'LEI', currentlyActive: true },
+]
+
+describe('getAdjudicationsPermissions', () => {
+  test.each`
+    user                                              | prisoner       | roles                   | viewVisible
```

Comment:
Really minor one on `viewVisible` and `editVisible`... sounds a bit strange, maybe should be `viewPermitted` or `editPermitted`?